CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
+
+### Release Notes
+* Features  
+  * Kubernetes 1.31 and 1.32 are now designated as [Long-Term Support (LTS)](https://learn.microsoft.com/azure/aks/long-term-support) versions.  
+  * Kubernetes 1.33 is available in Preview. A full matrix of supported add-ons and components is published at the [AKS versions page](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-1330).
+  * Network-isolated clusters are now Generally Available, including GA support for CNIv1 → Cilium and Cilium → Overlay upgrade paths.
+
+* Bug Fixes  
+  * Retained `kube-proxy` on add-ons v2 during Cilium migrations to prevent transient service-connectivity loss.  
+  * Updated the Windows GPU device-plugin to `v0.0.19`, mitigating CVE-2025-22871.  
+  * Switched the static-egress-gateway init container to the pre-built image `v0.0.20`, eliminating failures in environments without Internet access.  
+  * Re-enabled the `ExtensionManager` on Overlay, unblocking Flux extension deployments.  
+  * Overlay manager now ignores maintenance windows that aren’t weekly.  
+  * Fixed failures triggered by duplicate tag keys that differed only by character case.
+
+* Behavior Changes  
+  * In-place upgrades are now supported when the cluster is using [Azure CNI Overlay (preview)](https://learn.microsoft.com/azure/aks/azure-cni-overlay).
+  * Overlay manager disregards non-weekly maintenance windows (see Bug Fixes).  
+  * Duplicate tag keys that differ only by case are now rejected at creation time.
+
+* Component Updates  
+  * Updated NAP Custom Resource Definitions to [`1.4.0-aks`](https://github.com/Azure/karpenter-provider-azure/blob/main/charts/karpenter-crd/templates/karpenter.azure.com_aksnodeclasses.yaml)
+    * Added zone-label support and image-status fields.  
+    * Introduced `v1beta1` API for AKSNodeClass; default CRs updated accordingly.  
+    * Deployment flow now consumes `vnetGUID` from Secret when present.  
+    * Clean-up of obsolete resource versions/labels; broader ConfigMap read permissions for maintenance-window detection.  
+  * Windows node images  
+    * Server 2019 Gen1 – [`17763.7240.250416`](vhd-notes\AKSWindows\2019\17763.7240.250416.txt)
+    * Server 2022 Gen1/Gen2 – [`20348.3561.250416`](vhd-notes\AKSWindows\2022\20348.3561.250416.txt)
+    * Server 23H2 Gen1/Gen2 – [`25398.1551.250416`](vhd-notes\AKSWindows\23H2\25398.1551.250416.txt)
```

Comment:
these files are not added in the repo, these will result in broken links. after adding those files, also need to use '/' instead of '\'

Reply:
Good catch adding them.